examples/federated_learning/yolov5_coco128_mistnet/README.md

```diff
@@ -0,0 +1,238 @@
+mkdir -p /data
+echo "" > /data/test.txt
+```
+
+Create data interface for ```EDGE2_NODE```.
+
+```
+mkdir -p /data
+echo "" > /data/test.txt
+```
+
+### Prepare Images
+
+This example uses these images:
+
+1. aggregation worker: ```kubeedge/sedna-example-federated-learning-mistnet-yolo-aggregato:v0.4.0```
+2. train worker: ```kubeedge/sedna-example-federated-learning-mistnet-yolo-client:v0.4.0```
+
+These images are generated by the script [build_images.sh](/examples/build_image.sh).
+
+### Create Federated Learning Job
+
+#### Create Dataset
+
+create dataset for `$EDGE1_NODE` and `$EDGE2_NODE`
+
+```bash
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: Dataset
+metadata:
+  name: "coco-dataset"
+spec:
+  url: "/data/test.txt"
+  format: "txt"
+  nodeName: $EDGE1_NODE
+EOF
+```
+
+```bash
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: Dataset
+metadata:
+  name: "coco-dataset"
+spec:
+  url: "/data/test.txt"
+  format: "txt"
+  nodeName: $EDGE2_NODE
+EOF
+```
+
+#### Create Model
+
+create the directory `/model` in the host of `$CLOUD_NODE` (download links [here]())
+
+```bash
+mkdir models
+mkdir models/pretrained/
+# put pretrained model on sedna/examples/federated_learning/yolov5_coco128_mistnet/models/pretrained/yolov5.pth
+```
+
+```
+mkdir -p /model
+echo "" > /model/yolo.pb
+```
+
+create model
+
+```bash
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: Model
+metadata:
+  name: "yolo-v5-model"
+spec:
+  url: "/model/yolo.pb"
+  format: "pb"
+EOF
+```
+
+### Create a secret with your S3 user credential. (Optional)
+
+```shell
+kubectl create -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+  annotations:
+    s3-endpoint: s3.amazonaws.com 
+    s3-usehttps: "1" 
+stringData: 
+  ACCESS_KEY_ID: XXXX
+  SECRET_ACCESS_KEY: XXXXXXXX
+EOF
+```
+
+#### Start Federated Learning Job
+
+```bash
+kubectl create -f - <<EOF
+apiVersion: sedna.io/v1alpha1
+kind: FederatedLearningJob
+metadata:
+  name: yolo-v5
+spec:
+  pretrainedModel: # option
+    name: "yolo-v5-pretrained-model"
+  transmitter: # option
+    ws: { } # option, by default
+    s3: # optional, but at least one
+      aggDataPath: "s3://sedna/fl/aggregation_data"
+      credentialName: mysecret
+  aggregationWorker:
+    model:
+      name: "yolo-v5-model"
+    template:
+      spec:
+        nodeName: $CLOUD_NODE
+        containers:
+          - image: kubeedge/sedna-example-federated-learning-mistnet-yolo-aggregator:v0.4.0
+            name: agg-worker
+            imagePullPolicy: IfNotPresent
+            env: # user defined environments
+              - name: "cut_layer"
+                value: "4"
+              - name: "epsilon"
+                value: "100"
+              - name: "aggregation_algorithm"
+                value: "mistnet"
+              - name: "batch_size"
+                value: "32"
+            resources: # user defined resources
+              limits:
+                memory: 8Gi
```

Comment:
huge memory request, suggest remove/replace it

Reply:
It needs 8g (at least 7g) in the experiments. We cannot remove/replace it (This is on the cloud node, so the memory resource should be enough)